pkg/vetter/util/util.go

```diff
@@ -481,3 +482,24 @@ func ConvertHostnameToFQDN(hostname string, namespace string) (string, error) {
 	// need to return Fully Qualified Domain Name
 	return hostname + "." + namespace + KubernetesDomainSuffix, nil
 }
+
+// Extract status port from the cmd arguments for a given container, or
```

Comment:
Golang has a way to auto generate docs by inspecting the comments above methods.

Normally these start will the name of the function i.e. // ProxyStatusPort extracts the status port for sidecar proxies...

Reply:
Ok, will add function name in front.